addons/portal/controllers/portal.py

```diff
@@ -196,6 +194,82 @@ def account(self, redirect=None, **post):
         response.headers['X-Frame-Options'] = 'DENY'
         return response
 
+    @http.route('/portal/attachment/add_data', type='json', auth='public')
+    def attachment_add_data(self, name, data, res_model, res_id, access_token=None, **kwargs):
```

Comment:
and for consistency?
```suggestion
    def attachment_add(self, name, data, res_model, res_id, access_token=None, **kwargs):
```

Reply:
Ok